### PR TITLE
DP-21657: Add text search filter to ma-dash/feedback.

### PR DESCRIPTION
**Pull request by serundeputy**
**Description:**
Add `search` field to `/admin/ma-dash/feedback` allow users to search comma separate list of strings.

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-21657

**To Test:**
* visit `/admin/ma-dash/feedback`
* Add some search strings to the new `Search` text field
  * for example: `/admin/ma-dash/feedback?search=fleece&watch_content=0` should return 2 results
  * and: `/admin/ma-dash/feedback?search=fleece%2C%20underwear&watch_content=0` should return 4 results 

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by serundeputy** [comment]
@adamcogbill @joeg8612 this is likely to conflict w/ https://github.com/massgov/openmass/pull/752 so we can either merge them together or fix merge conflicts once one or the other gets merged.

**Comment by tom-fleming** [comment]
@serundeputy Looking over everything else as well, but the main thing I'm seeing so far is that there are a couple small CodeSniffer issues that need to be addressed. See https://app.circleci.com/pipelines/github/massgov/openmass/8671/workflows/d3d009d7-cfb0-4a18-aa06-3a256c8ab7e9/jobs/35420

**Comment by tom-fleming** [comment]
Hi @serundeputy It looks like we need two things to happen:
1. If you merge in the latest changes from develop, we merged in a memory limit increase that should overcome the Behat errors Circle is throwing
2. To fix the Danger alerts, you will need to add a changelog. You can see some examples here and instructions in the template.yml file: https://github.com/massgov/openmass/tree/develop/changelogs

File: changelogs/DP-21657.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# You may add more than 1 description & issue for each type using the following format:
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Type:
```
Comment (tom-fleming) [inline changelogs/DP-21657.yml]:
```suggestion
Added:
```

**Pull request merged**